src/components/vault/EntryDetails.js

```diff
@@ -39,10 +39,10 @@ import { useTranslations } from '../../hooks/i18n';
 
 const ENTRY_ATTACHMENT_ATTRIB_PREFIX = Entry.Attributes.AttachmentPrefix;
 const FIELD_TYPE_OPTIONS = [
-  { type: EntryPropertyValueType.Text, title: 'Text (default)', icon: 'italic' },
-  { type: EntryPropertyValueType.Note, title: 'Note', icon: 'align-left' },
-  { type: EntryPropertyValueType.Password, title: 'Password', icon: 'key' },
-  { type: EntryPropertyValueType.OTP, title: 'OTP', icon: 'time' }
+  { type: EntryPropertyValueType.Text, title: 'text', icon: 'italic' },
+  { type: EntryPropertyValueType.Note, title: 'note', icon: 'align-left' },
+  { type: EntryPropertyValueType.Password, title: 'password', icon: 'key' },
+  { type: EntryPropertyValueType.OTP, title: 'otp', icon: 'time' }
```

Comment:
Perhaps `title` should be changed to something like `i18n` to indicate that it's a key used for translations..?